R/tag.R

```diff
@@ -241,3 +241,21 @@ tag_markdown <- function(x) {
   x$val <- markdown_if_active(x$val, x)
   tag_value(x)
 }
+
+#' @export
+#' @rdname roxy_tag
+tag_markdown_with_sections <- function(x) {
+  x$val <- markdown_if_active(x$val, x, sections = TRUE)
+  if (length(x$val) == 1 && x$val[[1]] == "") {
+    roxy_tag_warning(x, "requires a value")
```

Comment:
Can you use the same test as above? And it needs to return, so that further processing doesn't occur.

Reply:
Hmmm, which test? As in `tag_value()`? `x$val` can be a vector now, so it needs a slightly different test.

Reply:
I think it would be natural to match `tag_examples()`; i.e. check `x$val == ""` before parsing the markdown.

Reply:
oh, ok, that makes sense.